tests/album-update-patch.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Album', () => {
+  let album;
+  beforeEach(async () => {
+    const { rows } = await db.query(
+      'INSERT INTO Albums (name, year) VALUES ($1, $2) RETURNING *',
+      ['Portrait With Firewood', 2018]
+    );
+
+    album = rows[0];
+  });
+
+  describe('PATCH /albums/{id}', () => {
+    it('updates the album and returns the updated record', async () => {
+      const { status, body } = await request(app)
+        .patch(`/albums/${album.id}`)
+        .send({ name: 'something else', year: 1992 });
+
+      expect(status).to.equal(200);
+
+      expect(body).to.deep.equal({
+        id: album.id,
+        name: 'something else',
+        year: 1992,
+        artistid: null,
+      });
+    });
+
```

Comment:
I would add additional tests here to test that the controller works if you update only the name or year. You are only testing when you update both